src/MoBi.R/Services/ProjectTask.cs

```diff
@@ -3,17 +3,30 @@
 using MoBi.Core.Domain.Model;
 using MoBi.Core.Serialization.ORM;
 using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
 using OSPSuite.R.Domain;
 
 namespace MoBi.R.Services
 {
    public interface IProjectTask
    {
-      MoBiProject GetProject(string fileName);
-      IReadOnlyList<string> GetModuleNames(MoBiProject moBiProject);
-      IReadOnlyList<string> GetSimulationNames(MoBiProject moBiProject);
-      IReadOnlyList<string> GetBuildingBlocksNamesFromModuleName(MoBiProject moBiProject, string moduleName);
-      IReadOnlyList<Simulation> GetSimulations(MoBiProject moBiProject);
+      MoBiProject LoadProject(string fileName);
+      IReadOnlyList<string> AllModuleNames(MoBiProject moBiProject);
+      IReadOnlyList<string> AllExpressionProfileNames(MoBiProject moBiProject);
+      IReadOnlyList<string> AllIndividualNames(MoBiProject moBiProject);
+      IReadOnlyList<string> AllSimulationNames(MoBiProject moBiProject);
+      IReadOnlyList<string> AllBuildingBlocksNamesFromModuleName(MoBiProject moBiProject, string moduleName);
+      IReadOnlyList<Simulation> AllSimulations(MoBiProject moBiProject);
+      IReadOnlyList<IndividualBuildingBlock> AllIndividuals(MoBiProject moBiProject);
+      IReadOnlyList<InitialConditionsBuildingBlock> AllInitialConditions(MoBiProject moBiProject);
+      IReadOnlyList<ParameterValuesBuildingBlock> AllParameterValues(MoBiProject moBiProject);
+      IReadOnlyList<InitialConditionsBuildingBlock> AllInitialConditionsFromModule(Module module);
+      IReadOnlyList<ParameterValuesBuildingBlock> AllParameterValuesFromModule(Module module);
+      Module ModuleByName(MoBiProject moBiProject, string name);
+      IndividualBuildingBlock IndividualBuildingBlockByName(MoBiProject moBiProject, string name);
+      InitialConditionsBuildingBlock InitialConditionBuildingBlockByName(Module module, string name);
+      ParameterValuesBuildingBlock ParameterValueBuildingBlockByName(Module module, string name);
```

Comment:
I bet we are going to need a `ModuleTask` soon because there are 4 methods here that don't use project at all. Next when you start loading modules from the file system, you will want to get initial conditions from the module without needing the project task.